js/src/framework.ts

```diff
@@ -10,16 +10,52 @@ import {
 import { Score, SpanTypeAttribute } from "@braintrust/core";
 import { BarProgressReporter, ProgressReporter } from "./progress";
 import pluralize from "pluralize";
+import { isEmpty } from "./util";
 
 export interface EvalCase<Input, Expected> {
   input: Input;
   expected?: Expected;
   metadata?: Metadata;
 }
 
+export type BaseExperiment<Input, Expected> = {
+  _type: "BaseExperiment";
+  _phantom?: [Input, Expected];
```

Comment:
is this used?

Reply:
Yeah it lets typescript believe the types are being used